resources/sdk/clisdkclient/templates/api.mustache

```diff
@@ -73,6 +73,11 @@ var {{operationId}}Cmd = &cobra.Command{
 		if len(queryParams) > 0 {
 			urlString = fmt.Sprintf("%v?", path)
 			for k, v := range queryParams {
+				if k == "varType" {
+					k = "type"
+					urlString += fmt.Sprintf("%v=%v&", url.QueryEscape(strings.TrimSpace(k)), url.QueryEscape(strings.TrimSpace(v)))
+					continue
```

Comment:
I don't think there's any need for lines 78 and 79, unless I'm missing something